README.md

```diff
@@ -45,7 +43,7 @@ This library is included by default in the openHAB [JavaScript Scripting add-on]
 
 ### Default Installation
 
-Install the openHAB [JavaScript Scripting add-on](https://www.openhab.org/addons/automation/jsscripting/), a version of this library will be automatically installed and available to ECMAScript 2024+ rules created using [File-Based Rules](#file-based-rules) or [Rules in Main UI](#rules-in-main-ui).
+Install the openHAB [JavaScript Scripting add-on](https://www.openhab.org/addons/automation/jsscripting/), a version of this library will be automatically installed and available to ECMAScript 2024+ rules created using [Rules in Main UI](#rules-in-main-ui) or [Rules created from Script Files](#rules-created-from-script-files).
 
```

Comment:
Won't it simply be available to all rules?

Reply:
> Should I rephrase?

Well, I just found it slightly confusing, implying that there are circumstances where it isn't available.

Reply:
Yes, I understand that, but the same would apply to YAML rules as "MainUI rules", for example, so couldn't it still be more generic similar to "will be automatically installed and available to ECMAScript 2024+ for all or some rules, depending on configuration."

Reply:
I have just updated the README.